packages/@orchidui-vue/src/Chip/OCChip.vue

```diff
@@ -1,11 +1,48 @@
 <template>
-  <span>
-    {{ label }}
+  <span class="rounded-full py-1 px-2 text-xs" :class="className">
+    <template v-if="label">{{ label }}</template>
+    <slot v-else />
   </span>
 </template>
-<script setup>
-import { defineProps } from "vue";
-defineProps({
-  label: String,
-});
-</script>
+<script>
+export default {
```

Comment:
We must finally decide which style we will use to write the Composition API or Options API components and redesign all components under one style.

Reply:
we will use composition api